docs/smart-rollup.md

```diff
@@ -7,7 +7,7 @@
 
 At first you should originate rollup with the provided alias for some implicit account:
 ```
-sudo -u tezos tezos-client originate smart rollup <ROLLUP_ALIAS> from <IMPLICIT_ACCOUNT_ALIAS>
+sudo -u tezos tezos-client originate smart rollup <ROLLUP_ALIAS> from <IMPLICIT_ACCOUNT_ALIAS> of kind <SMART_ROLLUP_KIND> of type <ROLLUP_PARAMETER_TYPE> with kernel <KERNEL>
```

Comment:
`ROLLUP_ALIAS` and `IMPLICIT_ACCOUNT_ALIAS` are more or less self-explanatory, but others should be documented with some example of use provided

Reply:
To be honest,  finding any examples on how exactly this binary should be used, is a total mess. I found a sort of documentation, which I provided link to